modules/ROOT/pages/release-notes.adoc

```diff
@@ -80,3 +80,29 @@ The optional "word-based" and default "start-with" search algorithms cannot be u
 Since "Word-based" search can no longer be used, its configuration has been removed. +
 The properties `bonita.platform.persistence.platform.enableWordSearch` and `bonita.platform.persistence.tenant.enableWordSearch` are removed from the configuration file `platform_engine/bonita-platform-community-custom.properties` when using the update tool, and they no longer have any effect. +
 If you used them to configure Bonita Runtime for "word-based" search and specified some objects to exclude using the bean `wordSearchExclusionMappings` in `platform_engine/bonita-platform-custom.xml`, you should remove this bean after updating since it is no longer used. The new "like-based" search is active on all objects that have API search methods.
+
+== Bug fixes
+
+=== Fixes in Bonita 2024.1-b1 (2024-02-13)
+
+==== Fixes in Bonita Runtime including Bonita Applications
+
```

Comment:
This line is duplicate:
```suggestion
```

Reply:
Copy paste issues you do not see because the line was copied form a different place without parentheses